tests/test_google_ads_page_limit.py

```diff
@@ -0,0 +1,68 @@
+from tap_tester import menagerie, connections, runner
+
+from base import GoogleAdsBase
+
+class TesGoogleAdstPagination(GoogleAdsBase):
+    """
+    Ensure tap can replicate multiple pages of data for streams that use query limit.
+    """
+    API_LIMIT = 1
```

Comment:
In addition to 1 can this test be ran with a more realistic api_limit without losing coverage? I think 1 is a good case to hit if we support it, but the default is much higher. At the very minimum I think we should always cover 2 just to verify we can sync multiple records per page.

Reply:
Updated `LIMIT` to 2.